website/docs/mutation.md

```diff
@@ -3,10 +3,6 @@ id: mutation
 title: Mutation
 ---
 
-`Feature State`: Gatekeeper version v3.7+ (beta)
```

Comment:
Might be good to update to ‘Feature state: Gatekeeper v3.10+ (stable)’ until older releases are out of support to signal folks to upgrade otherwise it’s hard to tell which release this feature went to stable.

Reply:
ack, done